recipes-mender/mender/files/mender.timer

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Timer for running Mender periodically 
+After=syslog.target
+
+[Timer]
+OnBootSec=1min
+OnUnitActiveSec=5min
+Unit=mender.service
+
+[Install]
+WantedBy=multi.user.target
```

Comment:
As discussed I think we definitely don't want this for Mender. Mender should either be a daemon or launched from some sort of on demand service daemon, such as xinetd (I think systemd also has some hooks for this). But most likely a standalone daemon.